File: modulo05/AulaFrasesDoDia/app/src/main/java/com/henriquejunqueira/aulafrasesdodia/MainActivity.java
package com.henriquejunqueira.aulafrasesdodia;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

import java.util.Random;

public class MainActivity extends AppCompatActivity {

    // Atributo
    String[] frases = {
            "Bom dia!",
            "Boa tarde!",
            "Boa noite!",
    };

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main), (v, insets) -> {
            Insets systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars());
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom);
            return insets;
        });
    }

    public void gerarFrase(View view){

        TextView texto = findViewById(R.id.textResultado);

        int numeroAleatorio = new Random().nextInt(3);

        String frase = frases[numeroAleatorio];

        texto.setText(frase);
    }

    public void exibirTodas(View view){

        TextView texto = findViewById(R.id.textResultado);

        String textoResultado = "";

        for(String frase : frases){
            textoResultado = textoResultado + frase + "\n";
        }

        texto.setText(textoResultado);

    }

}

File: modulo02/PrimeiroProjeto/app/src/main/java/com/henriquejunqueira/primeiroprojeto/MainActivity.kt
package com.henriquejunqueira.primeiroprojeto

import android.os.Bundle
import android.view.View
import android.widget.TextView
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import java.util.Random

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
            insets
        }
    }

    // função que será executada quando for clicado/pressionado o botão de sorteio
    fun sortear(view: View){

        // println("foi clicado!")

        // encontra um item de interface por ID e salva na variável "textoResultado"
        val textoResultado = findViewById<TextView>(R.id.text_resultado)

        // gera números de 0 a 10, o 11 não entra
        val numeroSorteado = Random().nextInt(11)

        // seta o valor do número sorteado na variável textoResultado que vai aparecer na tela
        textoResultado.setText("número gerado $numeroSorteado")

    }

}